src/jdk.hotspot.agent/bsd/native/libsaproc/ps_proc.c

```diff
@@ -357,7 +373,7 @@ static bool read_lib_info(struct ps_prochandle* ph) {
   for (i = 0; i < cnt; i++) {
     kve = &freep[i];
     if ((kve->kve_flags & KVME_FLAG_COW) &&
-        kve->kve_path != NULL &&
+        kve->kve_path[0] != '\0' &&
```

Comment:
What is the reasoning behind this change?

Reply:
kve_path is an array, thus can t be NULL

Reply:
Thanks.  Would 'strnlen(kve->kve_path, PATH_MAX) > 0' be more readable?

Reply:
No preference nor opinion here, same to me but will commit nevertheless.